xml/Microsoft.Extensions.Http.Logging/IHttpClientLogger.xml

```diff
@@ -1,139 +1,145 @@
+      <MemberSignature Language="DocId" Value="M:Microsoft.Extensions.Http.Logging.IHttpClientLogger.LogRequestFailed(System.Object,System.Net.Http.HttpRequestMessage,System.Net.Http.HttpResponseMessage,System.Exception,System.TimeSpan)" />
+      <MemberSignature Language="VB.NET" Value="Public Sub LogRequestFailed (context As Object, request As HttpRequestMessage, response As HttpResponseMessage, exception As Exception, elapsed As TimeSpan)" />
+      <MemberSignature Language="F#" Value="abstract member LogRequestFailed : obj * System.Net.Http.HttpRequestMessage * System.Net.Http.HttpResponseMessage * Exception * TimeSpan -&gt; unit" Usage="iHttpClientLogger.LogRequestFailed (context, request, response, exception, elapsed)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; void LogRequestFailed(System::Object ^ context, System::Net::Http::HttpRequestMessage ^ request, System::Net::Http::HttpResponseMessage ^ response, Exception ^ exception, TimeSpan elapsed);" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>Microsoft.Extensions.Http</AssemblyName>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="context" Type="System.Object">
+          <Attributes>
+            <Attribute>
+              <AttributeName Language="C#">[System.Runtime.CompilerServices.Nullable(2)]</AttributeName>
+              <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.Nullable(2)&gt;]</AttributeName>
+            </Attribute>
+          </Attributes>
+        </Parameter>
+        <Parameter Name="request" Type="System.Net.Http.HttpRequestMessage" />
+        <Parameter Name="response" Type="System.Net.Http.HttpResponseMessage">
+          <Attributes>
+            <Attribute>
+              <AttributeName Language="C#">[System.Runtime.CompilerServices.Nullable(2)]</AttributeName>
+              <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.Nullable(2)&gt;]</AttributeName>
+            </Attribute>
+          </Attributes>
+        </Parameter>
+        <Parameter Name="exception" Type="System.Exception" />
+        <Parameter Name="elapsed" Type="System.TimeSpan" />
+      </Parameters>
+      <Docs>
+        <param name="context">The context object that was previously returned by <see cref="M:Microsoft.Extensions.Http.Logging.IHttpClientLogger.LogRequestStart(System.Net.Http.HttpRequestMessage)" />.</param>
+        <param name="request">The HTTP request message that was sent.</param>
+        <param name="response">If available, the HTTP response message that was received, and `null` otherwise.</param>

```

Comment:
Backticks need to be replaced with `<c>`. (Here and elsewhere.)

Reply:
```suggestion
        <param name="response">If available, the HTTP response message that was received, and <see langword="null"/> otherwise.</param>

```